rest/taskana-rest-spring/src/test/java/pro/taskana/rest/TaskanaEngineControllerIntTest.java

```diff
@@ -49,9 +47,8 @@ void testClassificationTypes() {
             HttpMethod.GET,
             restHelper.defaultRequest(),
             ParameterizedTypeReference.forType(List.class));
-    assertTrue(response.getBody().contains("TASK"));
-    assertTrue(response.getBody().contains("DOCUMENT"));
-    assertFalse(response.getBody().contains("UNKNOWN"));
+    assertThat(response.getBody()).containsOnly("TASK", "DOCUMENT");
+    assertThat(response.getBody()).doesNotContain("UNKNOWN");
```

Comment:
The main benefit of using containsOnly is that we don't have to do additional checks like this one. Since the list contains **only** TASK and DOCUMENT